.packit.yaml

```diff
@@ -13,9 +10,6 @@ actions:
     - wget https://src.fedoraproject.org/rpms/p11-kit/raw/rawhide/f/trust-extract-compat
   get-current-version:
     - "git describe --abbrev=0"
-  create-archive:
-    - "wget https://github.com/p11-glue/p11-kit/releases/download/$PACKIT_PROJECT_VERSION/p11-kit-$PACKIT_PROJECT_VERSION.tar.xz"
-    - "wget https://github.com/p11-glue/p11-kit/releases/download/$PACKIT_PROJECT_VERSION/p11-kit-$PACKIT_PROJECT_VERSION.tar.xz.sig"
```

Comment:
> * create-archive is not used for propose_downstream

The current workflow is that, after uploading a release tarball along with the GPG signature, I open an issue to ask packit-service to propose-update, where I thought `create-archive` is used. Is it not the case?

Reply:
oh, I feel so silly now, you are correct, part of the downstream process is to upload tarballs

sorry about this, will drop that hunk

Reply:
The workflow you described sounds fine. Btw, cockpit team is planning to use something similar - we're discussing with them what would be the most convenient way to do when an archive is created outside packit: https://github.com/packit/packit-service/issues/1017